commands/new_function.go

```diff
@@ -25,6 +25,7 @@ var (
 
 func init() {
 	newFunctionCmd.Flags().StringVar(&language, "lang", "", "Language or template to use")
+	newFunctionCmd.Flags().StringVarP(&schemaVersion, "version", "v", defaultSchemaVersion, "The version to use for YAML stack file")
```

Comment:
i'm not sure about `--version` as we have `version`.  I wonder whether `schema` might offer better separation & meaning?

Is a flag even needed?  Should all new function be created at the default at the time they are created?  We definitely need to be able to react differently when reading a given version but will (or do) people want to explicitly set it to an earlier version when creating a new function?

Reply:
These are good points. I'll remove the flag.